compiler/rustc_target/src/spec/apple_base.rs

```diff
@@ -1,8 +1,46 @@
 use std::{borrow::Cow, env};
 
-use crate::spec::{cvs, FramePointer, LldFlavor, SplitDebuginfo, TargetOptions};
+use crate::spec::{
+    cvs, FramePointer, LinkArgs, LinkerFlavor, LldFlavor, SplitDebuginfo, TargetOptions,
+};
+
+fn pre_link_args(os: &'static str, arch: &'static str, target_abi: Option<&str>) -> LinkArgs {
```

Comment:
```suggestion
fn pre_link_args(os: &'static str, arch: &'static str, abi: &'static str) -> LinkArgs {
```
Same as in `fn opts`.